pkg/classification/interfaces/interfaces.go

```diff
@@ -228,15 +254,29 @@ func (classifier *Classifier) Classify(data detections.Detection) (*ClassifiedIn
 	}, nil
 }
 
-func (classifier *Classifier) FindMatchingRecipeUrl(detectionURL string) (*RecipeURLMatch, error) {
+func (classifier *Classifier) FindMatchingRecipeUrl(detectionURL string) (*RecipeURLMatch, *classify.ClassificationDecision, error) {
```

Comment:
Ideally this func wouldn't care about classification and would only care about matching URLs. 
Suggestion to add an ignored or excluded `Boolean` to `RecipeURLMatch`, which we can then use to classify the URL in the calling function above, and simplify the concerns of this func. What do you think?